Docs/source/running_cpp/pwfa.rst

```diff
@@ -123,6 +123,12 @@ Plotfiles and snapshots
     The diagnostics results are analysed and stored in snapshots at each time step and so it is best to make sure that the run does not end before filling the final snapshot.
 
 
+    .. note::
+
+       You may find example python scripts (in `Tools <Tools/PostProcessing/RunsScan/>_`) that do the analysis for convergence scans of the WarpX multi stage plasma wakefield acceleration configuration simulations.
```

Comment:
Maybe just verbatim:
```suggestion
       You may find example python scripts in Tools (``Tools/PostProcessing/RunsScan/``) that do the analysis for convergence scans of the WarpX multi stage plasma wakefield acceleration configuration simulations.
```
Otherwise:
```suggestion
       You may find example python scripts (in `Tools <https://github.com/ECP-WarpX/WarpX/tree/development/Tools/PostProcessing/RunsScan/>`__) that do the analysis for convergence scans of the WarpX multi stage plasma wakefield acceleration configuration simulations.
```

Reply:
Ah that works better yes! Thank you @ax3l 😁